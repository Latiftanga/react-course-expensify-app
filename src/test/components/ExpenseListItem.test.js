import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import expenses from '../fixtures/expenses';
import ExpensListItem from '../../components/ExpenseListItem';

test('should render expense list item', () => {
    const wrapper = shallow(<ExpensListItem key={expenses[1].id} {...expenses[1]}/>);
    expect(toJSON(wrapper)).toMatchSnapshot();
})
