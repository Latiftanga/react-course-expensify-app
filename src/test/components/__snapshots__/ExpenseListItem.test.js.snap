// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list item 1`] = `
<div>
  <Link
    to="2/edit"
  >
    <h3>
       
      Rent
       
    </h3>
  </Link>
  <p>
     
    109500
    - 
    -345600000
     
  </p>
</div>
`;
